kubernetes-maven-plugin/doc/src/main/asciidoc/inc/build/_configuration.adoc

```diff
@@ -66,25 +66,10 @@ endif::[]
 
 A provided `<from>` takes precedence over the name given here. This tag is useful for extensions of this plugin.
 
-| <<build-healthcheck, *healthCheck*>>
```

Comment:
This configuration is still used (DockerHealthCheckEnricher)

Reply:
Why do you need docker health checks when we have the same thing in kubernetes?

Reply:
I'm not saying that this feature is necessary or not (I haven't analyzed that so I don't have an opinion).
I'm just saying that this configuration is used in parts of the code.

Reply:
looks like you're right. We read this option and convert it to readiness/liveness probes